from_cpython/Python/mystrtoul.c

```diff
@@ -92,7 +92,7 @@ static int digitlimit[] = {
 **              exceptions - we don't check for them.
 */
 unsigned long
-PyOS_strtoul(register char *str, char **ptr, int base)
+PyOS_strtoul(register const char *str, char **ptr, int base)
```

Comment:
oh what, that's weird; in that case I think doing it the way you originally had it makes sense :/